portfolio/src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="loadComments()">
```

Comment:
`loadComments()` here would load comments on page load, before a user has had the chance to make a choice regarding how many comments they would like to see.  This behavior is fine and is pretty common to load a default set.

Only thing is the servlet (which Github listed first and I then reviewed first) sounded like userChoice was required and only defaults to 3 on error scenarios (eg. if user picked a non-numeric value). In the case here, a blank userChoice is a legit action since it's used to pull the default comments set.  Maybe a quick fix for this would be to rename userChoice to someting more generic like "comments_quantity", which could be used both by the user actually picking something, or by the system's default behavior.